hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java

```diff
@@ -134,7 +149,8 @@ public void checkSolvency(
             @NonNull final Fees fees,
             // This is to match mono and pass HapiTest. Should reconsider later.
             // FUTURE ('#9550')
-            final boolean ingestCheck)
+            final boolean ingestCheck,
+            final boolean checkOfferedFee)
```

Comment:
Would love to get rid of these booleans somehow...probably just wishful thinking

Reply:
If we want to behave the same as existing system, we need them. I will add an Enum for clarity, but I think we cant remove those conditions. 🤔

Reply:
Changed to enums @mhess-swl . Thank you